src/m_class.c

```diff
@@ -147,6 +147,11 @@ EXTERN void pd_setinstance(t_pdinstance *x)
     pd_this = x;
 }
 
+EXTERN t_pdinstance *pd_getinstance(void)
```

Comment:
i guess this `EXTERN` in the implementation can be omitted (since it's already used in the function declaration)

Reply:
Ah, yes. I just blindly copied it from the other functions and you are completely right that it is redundant. I just remembered that you already removed `EXTERN` from function definitions on the develop branch (https://github.com/pure-data/pure-data/commit/7a59ac2d59901ae05b96531e7ef8363d839a9797).